src/drivers/frsky_telemetry/frsky_data.c

```diff
@@ -195,6 +201,13 @@ void frsky_update_topics()
 	if (updated) {
 		orb_copy(ORB_ID(vehicle_global_position), vehicle_global_position_sub, global_pos);
 	}
+
+	/* get a local copy of the global position data */
```

Comment:
Nitpick : Should be "GPS data" to avoid confusion with the global position estimate.